docs/src/references/mqtt-api.md

```diff
@@ -647,6 +647,66 @@ tedge mqtt pub -r te/device/main///cmd/config_snapshot '{
 }'
 ```
 
+## Signals
+
+Signals are stateless, making them ideal for one-shot requests.
+The topic scheme for signals can be visualized using the diagram below.
+
+<p align="center">
+
+```mermaid
+graph LR
+  te --/--- identifier --/--- cmd
+  subgraph root
+    te
+  end
+
+  subgraph identifier
+    identifier2["&lt;identifier&gt;"]
+  end
+
+  subgraph command
+    direction LR
+    sig --/--- signal_type["&lt;signal_type&gt;"] --/--- action["&lt;action&gt;"]
+  end
```

Comment:
The reference to a subgraph must use the name of that subgraph. Otherwise mermaid displays two independent graphs.

```suggestion
graph LR
  te --/--- identifier --/--- signal
  subgraph root
    te
  end

  subgraph identifier
    identifier2["&lt;identifier&gt;"]
  end

  subgraph signal
    direction LR
    sig --/--- signal_type["&lt;signal_type&gt;"] --/--- action["&lt;action&gt;"]
  end
```

Reply:
Fixed in https://github.com/thin-edge/thin-edge.io/pull/3761/commits/8f28b066c52015fbf18d04db21ee09a841e783c9